examples/SkeletalCartesianLambdaIntroduction.g

```diff
@@ -29,12 +29,22 @@ elf := CartesianLambdaElimination( S, R, lf );
 #! |3| → |2|
 elf = f;
 #! true
-L := MorphismsOfExternalHom( S, R );
-#! [ |3| → |2|, |3| → |2|, |3| → |2|, |3| → |2|,
-#!   |3| → |2|, |3| → |2|, |3| → |2|, |3| → |2| ]
-Li := List( L, phi -> CartesianLambdaIntroduction( phi ) );
-#! [ |1| → |8|, |1| → |8|, |1| → |8|, |1| → |8|,
```

Comment:
Why is hiding this output desirable? This is also a documentation file, not merely a test file.

Reply:
The output is not hidden, its even expanded by the Display in line 33.